src/EnergyPlus/UnitarySystem.cc

```diff
@@ -16021,7 +16021,7 @@ namespace UnitarySystems {
             SpeedNum = int(Par[5]);
             FanOpMode = int(Par[6]);
 
-            HeatingCoils::CalcMultiStageElectricHeatingCoil(state, CoilIndex, SpeedRatio, CycRatio, SpeedNum, FanOpMode, QActual, SuppHeat);
+            HeatingCoils::CalcMultiStageGasHeatingCoil(state, CoilIndex, SpeedRatio, CycRatio, SpeedNum, FanOpMode);
```

Comment:
I was wondering why the CalcMultiStageElectricHeatingCoil passed a SuppHeat argument and see this. Why doesn't this happen regardless if the coil is a heating coil or supplemental heating coil? I think these staged coils (all staged coils) should always use MSHPMassFlowRateLow and MSHPMassFlowRateHigh for the calculations. So if cycling fan the air flow used by the model is the top end flow, and if constant fan the bottom end flow. Something feels off here.

                // for cycling fan, reset mass flow to full on rate
                if (FanOpMode == CycFanCycCoil) AirMassFlow /= PartLoadRat;
                if (FanOpMode == ContFanCycCoil) {
                    if (!SuppHeat) {
                        AirMassFlow = state.dataHVACGlobal->MSHPMassFlowRateLow;
                    }
                }

Reply:
In the PR #9414 I [removed MSHPMassFlowRateHigh and MSHPMassFlowRateLow](https://github.com/NREL/EnergyPlus/pull/9414/files#diff-8b37b587f8203d470dc38424b1d945e28d1deb62a1342bfdaa1643b451844270L2054-L2055) in [other calculations](https://github.com/NREL/EnergyPlus/pull/9414/files#diff-8b37b587f8203d470dc38424b1d945e28d1deb62a1342bfdaa1643b451844270L2147-R2191). You will see they're equivalent if you write down all the previous equations and offset against variables. And for supplemental coil, isn't it good enough to [pass the inlet airflow rate](https://github.com/NREL/EnergyPlus/pull/9414/files#diff-8b37b587f8203d470dc38424b1d945e28d1deb62a1342bfdaa1643b451844270R2162) for constant fan?

Reply:
I did it in this way because `MSHPMassFlowRateLow ` and `MSHPMassFlowRateHigh` are basically only set for main coils, I don't think it's worth creating two additional variables and setting proper values specifically for supplemental coils if there's no obvious reason to do so.

Reply:
Current approach is consistent with single speed electric coil (the same function is called for both supplemental coils and main heating coils):
https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/HeatingCoils.cc#L1978-L1987.
When it's constant fan, simply pass the InletAirMassFlowRate otherwise divide it by part load ratio. 

Even though they're consistent, I still got questions: The part load ratio being divided here is the supplemental coil part load ratio instead of the main coil part load ratio. Is it correct? It means if the main heating coil runs 100%, while the PLR of supp coil <1, then the enthalpy/temperature at outlet is calculated with airflow rate (inlet airflow/suppcoilPLR), is inlet airflow the full airflow rate?(is it determined by main coil?). Another case is that the main coil is not enabled (eg. outdoor temperature too low), then the backup coil will be the only heating source, will the airflow be determined by its PLR? I think the only doubt from me is to make sure the part load ratio being divided makes sense for both situations.